### ISSUE TITLE
Trying to add a feed, getting strange small empty message

### ISSUE THREAD

**Issue by TheScrawl** [jangernert/FeedReader#702]
I'm trying to add http://www.radionz.co.nz/rss/political.xml to my feeds, assinging it to the politics category. When I try to add it, it loads for a second then a small empty messagebox pops up. The feed is not added.

[screenshot](https://i.imgur.com/VOsZKA5.png)

Im running Manjaro with XFCE on an Acer Aspire R3-471TG

**Comment by EmpireJones** [comment]
Here's another example of a feed that does this:
https://code.fb.com/category/networking-traffic/rss/

Ubuntu 18.04.1 LTS

**Comment by bilelmoussaoui** [comment]
Which backend plugin are you guys using? Local RSS?

**Comment by EmpireJones** [comment]
Yeah, I was using Local RSS

**Comment by sm-Fifteen** [comment]
I'm getting the same issue on local RSS, though it seems to be somewhat inconsistant with what URLs do and don't get that issue (http://www.radionz.co.nz/rss/political.xml and https://code.fb.com/category/networking-traffic/rss/ work just fine on my end). If I can find an RSS feed where that issue occurs, it seems to be reproductable every time, even acter closing the Feedreader window or the entire application (Ctrl+Q).

The console logs don't show anything useful, both working and non-working URLs will log `addFeed: ID = 16f90e18-fe3a-4154-a673-d276d974856d`, except working feeds will follow up with a sync operation and download the RSS data while those subject to this issue won't and the empty tooltip box from the screenshot in the issue will occur.

I have tried querying the same services running under different hostnames (including directly via their IP) and I found out that using different hostnames can workaround the issue, but I don't think it's just something to do with the strings since just switching the protocol from HTTP to HTTPS and vice versa or adding the port number at the end of the hostname doesn't fix the issue. It's also not a problem with the service sending invalid data or being unresponsive, since I'm already subscribed to that URL and it's working as intended, I just can't remove the feed and then add it back.
